Faithlife.PortableContrib.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Faithlife.PortableContrib</id>
+    <version>$version$</version>
+    <authors>Faithlife Corporation</authors>
+    <owners>Faithlife Corporation</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Adapters and implementations of common types not otherwise available to Portable Class Libraries.</description>
+    <copyright>Copyright 2015 Faithlife Corporation</copyright>
+    <dependencies>
+    </dependencies>
+  </metadata>
+  <files>   
+    <file src="src\Faithlife.PortableContrib\bin\Portable\$configuration$\Faithlife.PortableContrib.*" target="lib\portable-net45+win+wp8\" />

```

Comment:
I copied it from what I did for System.Data.SQLite, but I forgot that was profile 78 and this is 111. I'll fix it.

This is a good reference for PCL profiles: http://plnkr.co/03ck2dCtnJogBKHJ9EjY